cs_api/cs_api/db/db.py

```diff
@@ -0,0 +1,63 @@
+import yaml
+
+from cs_api.server import db
+from cs_api import constants as const
+from dropbox_rclone import dropbox_reading
+
+# Because models need to be imported after db gets imported
+from cs_api.db.models import *
+
+
+def get_data_types():
+    """
+    Get the data types from the database
+    :return: list of data types
+    """
+    return [data_type.data_type for data_type in DataType.query.all()]
+
+
+def get_tect_types():
+    """
+    Get the tectonic types from the database
+    :return: list of tectonic types
+    """
+    return sorted([tect_type.tect_type for tect_type in TectType.query.all()])
+
+
+def get_grid_spacings():
+    """
+    Get the grid spacings from the database
+    :return: list of grid spacings
+    """
+    return sorted(
+        [grid_spacing.grid_spacing for grid_spacing in GridSpacing.query.all()]
+    )
+
+
+def get_scientific_versions():
```

Comment:
Whats the scientific version?

Reply:
Scientific version is something specific to cybershake runs, we currently have 18.6 and 20.4.1. This might need to be revisited as a filter based on any non cybershake simulation runs being used.